src/ir/ty.rs

```diff
@@ -104,9 +104,13 @@ impl Type {
         Self::new(Some(name), None, kind, false)
     }
 
-    pub fn is_integer_literal(&self) -> bool {
+    pub fn is_integer(&self, ctx: &BindgenContext) -> bool {
         match *self.kind() {
             TypeKind::Int(..) => true,
+            TypeKind::ResolvedTypeRef(inner) |
+            TypeKind::Alias(_, inner) => {
```

Comment:
Yeah, on Mac you need to `brew install llvm38` at least (and then maybe configure paths and all, I don't remember correctly)